packages/app-store/routing-forms/lib/getQueryBuilderConfig.ts

```diff
@@ -1,23 +1,35 @@
-import type { QueryBuilderUpdatedConfig, RoutingForm } from "../types/types";
+import { AttributeType } from "@calcom/prisma/client";
+
+import type { RoutingForm, Attribute } from "../types/types";
 import { FieldTypes } from "./FieldTypes";
-import { InitialConfig } from "./InitialConfig";
+import { AttributesInitialConfig, FormFieldsInitialConfig } from "./InitialConfig";
 import { getUIOptionsForSelect } from "./selectOptions";
 
-export function getQueryBuilderConfig(form: Pick<RoutingForm, "fields">, forReporting = false) {
-  const fields: Record<
-    string,
-    {
-      label: string;
-      type: string;
-      valueSources: ["value"];
-      fieldSettings: {
-        listValues?: {
-          value: string;
-          title: string;
-        }[];
-      };
-    }
-  > = {};
+type RaqbConfigFields = Record<
+  string,
+  {
+    label: string;
+    type: string;
+    valueSources: ["value"];
+    fieldSettings: {
+      listValues?: {
+        value: string;
+        title: string;
+      }[];
+    };
+  }
+>;
+
+export type FormFieldsQueryBuilderConfigWithRaqbFields = typeof FormFieldsInitialConfig & {
+  fields: RaqbConfigFields
+};
+
+export type AttributesQueryBuilderConfigWithRaqbFields = typeof AttributesInitialConfig & {
+  fields: RaqbConfigFields
+};
```

Comment:
They are same right now but could be different. So, it makes sense to keep them different entitities.